src/routes/simulateur/Forms/NumberForm.tsx

```diff
@@ -0,0 +1,49 @@
+import { component$, Signal, useComputed$, QRL, $ } from "@qwik.dev/core";
+import { steps } from "../steps";
+import { Answer } from "..";
+
+interface NumberProps {
+  add: QRL<(answer: Answer, nextOption?: string) => any>;
+  current: Signal<string>;
+}
+
+export default component$<NumberProps>(({ current, add }) => {
+  const step = useComputed$(() => steps[current.value]);
+
+  const onSubmit = $((form: HTMLFormElement) => {
+    if (step.value.type !== 'number') return;
+    const formData = new FormData(form);
+    const formValue: Record<string, string> = {}
+    for (const [key, value] of formData.entries()) {
+      const label = step.value.props.fields[key].label;
+      formValue[key] = `${label}: ${value}`;
```

Comment:
```suggestion
      formValue[key] = value`;
```

Reply:
Sure let's keep what's working. I'll ask in the qwik discord how to achieve that